src/core/components/choice-card/styles.ts

```diff
@@ -29,6 +29,16 @@ export const flexContainer = css`
 	}
 `
 
+export const gridContainer = ({columns} : {columns : number}) => css`
+	width: 100%;
+
+	${from.mobileLandscape} {
+		display: grid;
```

Comment:
Maybe we can put this block inside a [`@supports(display: grid) { ... }`](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) block. This way we can think about how it would look in browsers that don't fully support CSS grid (IE11).